modules/web/src/app/node-data/template.html

```diff
@@ -249,12 +249,14 @@
         </mat-card-title>
       </mat-card-header>
 
-      <mat-checkbox *ngIf="!isDialogView()"
-                    [name]="Controls.EnableClusterAutoscalingApp"
-                    [formControlName]="Controls.EnableClusterAutoscalingApp">Enable Cluster Autoscaler Application
+      <div fxLayoutAlign=" center">
+        <mat-checkbox *ngIf="!isDialogView()"
```

Comment:
This `*ngIf` should be on parent div otherwise tooltip will be visible in dialog view.